django_project/frontend/api_views/national_statistic.py

```diff
@@ -59,14 +62,8 @@ def get_statistics(self, request):
         total property area available for
         owned species (national).
         """
-        organisation_id = request.session.get(
-            'current_organisation_id'
-        )
 
-        national_properties = Property.objects.filter(
-            property_type__name='national',
-            organisation=organisation_id
-        )
+        national_properties = Property.objects.filter()
```

Comment:
@tinashechiraya is this correct? It seems that you haven't added any filters.

Reply:
@dimasciput Yes I talked to Luna she informed me that for a national level the user needs to be able to get data country wide regardless of organisation or property type